util/collate/charset.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collate
+
+import (
+	"strings"
+
+	"github.com/pingcap/parser/charset"
+)
+
+var (
+	enableCharsetFeat bool
+)
+
+// EnableCharsetFeat enables the charset feature.
+func EnableCharsetFeat() {
```

Comment:
Do we need to change it to `EnableNewCharset`? This matches the collation feature.